plugins/index.js

```diff
@@ -82,15 +75,15 @@ const describePlugin = (manifest) => (
 export function loadPlugin(manifest) {
     return new Promise((resolve, reject) => {
         // Don't load it again if previously loaded
-        const oldManifest = loadedPlugins[manifest.id];
+        const oldManifest = store.getState().plugins.plugins[manifest.id];
```

Comment:
Hey @Qovaros, you'll hate me for this but wondering if you can rename the `plugins` store to `manifests` as then we won't have this stutter, then it'll become
`store.getState().plugins.manifests[manifest.id];` which looks much nicer :)

Reply:
Yeah, sure, why not. It is a good idea.